src/pytest_plugins/solc/solc.py

```diff
@@ -90,6 +100,20 @@ def pytest_configure(config: pytest.Config):
         )
     config.solc_version = solc_version_semver  # type: ignore
 
+    # test whether solc_version matches actual one
```

Comment:
Perhaps you could add a comment why you obtain the version this way? 

I.e. It's due to how the static filler runs `solc`: https://github.com/ethereum/execution-spec-tests/blob/c91549cf31c503d5ea9b3037093449dd0bb4d146/src/ethereum_test_specs/static_state/common/compile_yul.py#L22